### PR TITLE
Bug/e UI 3505 crud aa

### PR DESCRIPTION
**Pull request by adnanakgun**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3505

### Change description ###

Applied fix to disable and enable collection remove button based on CRUD permissions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/shared/components/palette/collection/write-collection-field.component.ts [hunk 1]
```diff
@@ -246,20 +246,18 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
...
+  isNotAuthorisedToUpdate(index) {
     if (this.isExpanded) {
       return false;
     }
-    // TODO: Reassess the logic around the id when we know what the behaviour should actually
-    // be as what was in place prevents creation of new items as it shows a readonly field
-    // rather than an writable component.
-    // const id = this.getControlIdAt(index);
-    // if (!!id) {
+    // Was reassesed as part of EUI-3505. There is still a caveat around CRT, but that was deemed an unlikely scenario
```
Comment (paul-graham) [inline src/shared/components/palette/collection/write-collection-field.component.ts:253]: CRD, I assume?

**Review by paul-graham — DISMISSED** [review]
All looks good to me. There was just one misspelling in a comment but I'm not going to be too picky about such things! 😄

**Review by udaydenduluri33 — APPROVED** [review]
LGTMa

**Review by paul-graham — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**